projects/plugins/jetpack/_inc/lib/core-api/wpcom-fields/post-fields-publicize-connections.php

```diff
@@ -6,12 +6,13 @@
  * { # Post Object
  *   ...
  *   jetpack_publicize_connections: { # Defined below in this file. See schema for more detail.
- *     id:           (string)  Connection unique_id
- *     service_name: (string)  Service slug
- *     display_name: (string)  User name/display name of user/connection on Service
- *     enabled:      (boolean) Is this connection slated to be shared to? context=edit only
- *     done:         (boolean) Is this post (or connection) done sharing? context=edit only
- *     toggleable:   (boolean) Can the current user change the `enabled` setting for this Connection+Post? context=edit only
+ *     id:              (string)  Connection unique_id
+ *     service_name:    (string)  Service slug
+ *     display_name:    (string)  User name/display name of user/connection on Service
+ *     profile_picture: (string) Profile picture of user/connection on Service
```

Comment:
The only change in this file is adding `profile_picture` to the endpoint schema.